layouts/partials/head.html

```diff
@@ -10,6 +10,8 @@
     <link rel="icon" type="image/png" sizes="16x16" href={{ "favicon-16x16.png" | absURL }}>
     <link rel="manifest" href={{ "site.webmanifest" | absURL }}>
     <link rel="mask-icon" href={{ "safari-pinned-tab.svg" | absURL }} color="#00416a">
+    <link rel="feed" type="application/rss+xml" href="./index.xml" title="{{ .Title }}" />
+    <link rel="alternate" type="application/rss+xml" href="./index.xml" title="{{ .Title }}" />
```

Comment:
I like that using `./index.html` will allow for feed auto-discovery of `/posts/index.xml`, `/tags/index.xml`, etc.

Should there be some additional logic would be useful here to make sure that we exclude, for example, `/pages/2` (which doesn't have an associated `index.xml` file)?

Reply:
ideally, yes, It'd be better to only include a link where present. I don't know how to do that.